pkg/formats/formats.go

```diff
@@ -77,3 +99,35 @@ func (f *Format) Type() string {
 	}
 	return ""
 }
+
+type FormatOpt struct {
+	FormatType    string `yaml:"format-type,omitempty" json:"format-type,omitempty"`
+	FormatVersion string `yaml:"format-version,omitempty" json:"format-version,omitempty"`
+	MimeFormat    string `yaml:"mime-format,omitempty" json:"mime-format,omitempty"`
+}
```

Comment:
Ah I see what you mean here.Well, we don'r really need these. The `format.Format` type can be overloaded to give you each of these parts. We just need to add the methods. For example:
```go
version := SPDX23JSON.Version()  // 2.3
standard := SPDX23JSON.Standard()  // spdx
encoding := SPDX23JSON.Encoding() // "json"
```
 
```